models/ios/vxlan_vtep/ios_vxlan_vtep.yaml

```diff
@@ -0,0 +1,97 @@
+module: ios_vxlan_vtep
+short_description: Resource module to configure VXLAN VTEP interface.
+description: This module provides declarative management of VXLAN VTEP interface on Cisco IOS network
+  devices.
+version_added: 1.0.0
```

Comment:
correct the version added to the latest version of the collection.

Reply:
Corrected